pkgs/tools/misc/coreutils/default.nix

```diff
@@ -53,7 +53,7 @@ stdenv.mkDerivation rec {
       ++ optional aclSupport acl.crossDrv
       ++ optional attrSupport attr.crossDrv
       ++ optionals selinuxSupport [ libselinux.crossDrv libsepol.crossDrv ]
-      ++ optional (stdenv.ccCross.libc ? libiconv)
+      ++ optional (stdenv.ccCross or {} ? libc.libiconv)
```

Comment:
I can't really remember, but I think the issue I originally ran into was that some part of that expression had a `null` instead of just being absent, so that's what I came up with. I don't think I had considered putting more than 0 dots on the rhs of the `?`, though. I will just go with your last suggestion, if it works.